boule/__init__.py

```diff
@@ -1,3 +1,8 @@
+# Copyright (c) 2019 The Boule Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
```

Comment:
Should we add an empty comment line here to separate this from the pylint configuration?

```suggestion
# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
#
```

Reply:
Or maybe remove the empty comment line before this. Either one would do.

Reply:
You are right. I like the empty line between the license notice and the `This code is part...` line. So we should add an empty comment line after that one.